process/project-lifecycle-documents/SLSA_graduation_stage.md

```diff
@@ -0,0 +1,72 @@
+  * Yes, all relevant information is included below.
+
+### List of project maintainers
+The project must have maintainers with a minimum of five different contributors from three different organizational affiliations.
+
+  * SLSA has active maintainers across many different organizations including Datadog, GitHub, Google, IBM, Intel, Kusari, New York University, Red Hat.
+    See https://github.com/slsa-framework/slsa/blob/main/MAINTAINERS.md
+
+### Sponsor
+Most projects will report to an existing OpenSSF Working Group, although in some cases a project may report directly to the TAC. The project commits to providing quarterly updates on progress to the group they report to.
+  * [Supply Chain Integrity WG](https://github.com/ossf/wg-supply-chain-integrity/tree/main)
+
+### Mission of the project
+The project must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be code needed to deliver OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+  * SLSA (pronounced "salsa") is a security framework, a checklist of standards and controls to prevent tampering, improve integrity, and secure packages and infrastructure. It’s how you get from "safe enough" to being as resilient as possible, at any link in the software supply chain.
+  * The SLSA specification is useful for both software producers and consumers: producers can follow SLSA’s guidelines to make their software supply chain more secure, and consumers can use SLSA to make decisions about whether to trust a software package.
+
+### Project adoption
+The project must be able to show adoption by multiple parties, which could be production deployments or substantial use by established open source communities, and demonstrate the value of that adoption to either the end users or the open source community.
+  * Since the release of SLSA 1.0 in April 2023, evidence of its adoption keeps growing. This includes support in build platform offerings from companies like [Google](https://cloud.google.com/build/docs/overview), [IBM](https://cloud.ibm.com/docs/devsecops?topic=devsecops-cd-devsecops-slsa), and [Red Hat](https://developers.redhat.com/products/trusted-software-supply-chain/overview), as well as support for [npm package provenance in GitHub](https://github.blog/security/supply-chain-security/introducing-npm-package-provenance/) and integration with Sigstore among others.
+
+### Release cadence
+The project must be able to show a consistent release cadence.
+  * https://github.com/slsa-framework/slsa/tags
+
+### Governance
+Projects must have documented project governance and be able to demonstrate that governance in action.
+  * https://github.com/slsa-framework/governance
+
+Have a defined and documented roadmap and annual goals for the project
+  * https://github.com/slsa-framework/slsa/projects?query=is%3Aopen
```

Comment:
At first I was skeptical, but this is actually not a bad way to indicate the high-level workstreams a TI is undertaking, especially for a larger project like SLSA!

Reply:
I think one could argue that we are missing specific target dates. We are aiming for SLSA 1.1 to be out by the end of the year but this isn't captured here but otherwise I think it's a pretty good view of what's going on.

Reply:
Should it be captured here just to meet the metric? The case is strong for graduated status, but we do want to make sure the "I"s are dotted, and "T"s are crossed.

Reply:
One way we might track our roadmap is by using GitHub milestones, as we've done [in the past](https://github.com/slsa-framework/slsa/milestones]. These specifically allow us to set due dates, and to associate specific issues and PR with a milestone to track progress.